tutorials/Python/Python-skills/using-jupyter-notebooks/Jup02-JupyterNotebook/Jup02-JupyterNotebook.md

```diff
@@ -51,36 +52,36 @@ following directions pertain directly to the set up for the 2018 Data Institute
 on Remote Sensing with Reproducible Workflows, however, you can adapt them to 
 the specific Python version and packages you wish to work with.  
 
-If you haven't yet created a Python 3.5 environment, you'll need to do 
+If you haven't yet created a Python 3.8 environment (released October 2019), you'll need to do 
```

Comment:
Awesome! Thank you for updating this to the latest version of Python! This tutorial will be super helpful for new users